gci/students.py

```diff
@@ -129,6 +130,39 @@ def get_issue_related_students():
                     break
 
 
+def get_pr_closing_issue(url):
+    r = requests.get(url)
+    result = re.search(
+        'closed this(?:\n| )+in(?:\n| )+<a href=\"/(.+)/pull/(?:[0-9]+)\">#' +
+        '([0-9]+)</a>',
+        r.text
+    )
+
+    if result is not None:
+        return {
```

Comment:
This should return a `MergeRequest` , which has a `.author` added in https://gitlab.com/gitmate/open-source/IGitt/merge_requests/159 ?

But also, this function should be in `IGitt`.

Reply:
Also note that issues can be re-opened, and have a second commit which closes them again, so there is no 1-1 relationship.